manifest.json

```diff
@@ -4,9 +4,9 @@
   "manifest_version": 1,
   "action": {
     "title": "Dragonbane Sheet",
-    "icon": "/apps/obr-dragonbane-sheet/sheet.svg",
+    "icon": "/icon.svg",
```

Comment:
This will need the whole url prefix (/apps/obr-dragonbane-sheet/) otherwise the icon won't load when the code is built and deployed.